metadata_service/proxy/atlas_proxy.py

```diff
@@ -116,22 +142,68 @@ def delete_owner(self, *, table_uri: str, owner: str) -> None:
         pass
 
     def add_owner(self, *, table_uri: str, owner: str) -> None:
-        pass
+        """
+        It simply replaces the owner field in atlas with the new string.
+        FixMe (Verdan): Implement multiple data owners and
+        atlas changes in the documentation if needed to make owner field a list
+        :param table_uri:
+        :param owner: Email address of the owner
+        :return: None, as it simply adds the owner.
+        """
+        entity, _ = self._get_table_entity(table_uri=table_uri)
+        entity.entity['attributes']['owner'] = owner
+        entity.update()
 
     def get_table_description(self, *,
                               table_uri: str) -> Union[str, None]:
-        pass
+        """
+        :param table_uri:
+        :return: The description of the table as a string
+        """
+        entity, _ = self._get_table_entity(table_uri=table_uri)
+        return entity.entity['attributes']['description']
 
     def put_table_description(self, *,
                               table_uri: str,
                               description: str) -> None:
-        pass
+        """
+        Update the description of the given table.
+        :param table_uri:
+        :param description: Description string
+        :return: None
+        """
+        entity, _ = self._get_table_entity(table_uri=table_uri)
+        entity.entity['attributes']['description'] = description
+        entity.update()
 
     def add_tag(self, *, table_uri: str, tag: str) -> None:
-        pass
+        """
+        Assign the tag/classification to the give table
+        API Ref: /resource_EntityREST.html#resource_EntityREST_addClassification_POST
+        :param table_uri:
+        :param tag: Tag/Classfification Name
+        :return: None
+        """
+        entity, _ = self._get_table_entity(table_uri=table_uri)
+        entity_bulk_tag = {"classification": {"typeName": tag},
+                           "entityGuids": [entity.entity['guid']]}
+        self._driver.entity_bulk_classification.create(data=entity_bulk_tag)
 
     def delete_tag(self, *, table_uri: str, tag: str) -> None:
-        pass
+        """
+        Delete the assigned classfication/tag from the given table
+        API Ref: /resource_EntityREST.html#resource_EntityREST_deleteClassification_DELETE
+        :param table_uri:
+        :param tag:
+        :return:
+        """
+        try:
+            entity, _ = self._get_table_entity(table_uri=table_uri)
+            guid_entity = self._driver.entity_guid(entity.entity['guid'])
+            guid_entity.classifications(tag).delete()
+        except Exception as ex:
```

Comment:
This could blind us from real failure. Is there some way to filter out this Exception that happens all the time?

Reply:
There could be multiple exceptions: 
- Tag Deleted: But there still some tags associated. Can't find this tag exception
- Last Tag Deleted: Different exceptions, sometimes it raises no tags associated, and sometimes it says can't find this tag. 

I will add a FixMe here, and will debug this issue once I'll improve the atlas client library. (planning to fork that one as the maintainer is not responsive)